### PR TITLE
fix(oEmbed): Bring iframe in line with the embed in the GUI

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Just like you have at
https://github.com/testing-library/testing-playground/blob/dc14e4ae6430876cdb0a36fda09486fdb3ae6d93/src/components/Embed.js#L37

### CODE REVIEW

File: src/lambda/oembed/oembed.js [hunk 1]
```diff
@@ -34,7 +34,8 @@ function handler(event, context, callback) {
     );
   }
 
-  const { url, referrer, maxwidth = 900, maxheight = 300 } = params;
+  const { url, referrer, maxwidth = '100%', maxheight = 300 } = params;
```
Comment (smeijer) [inline src/lambda/oembed/oembed.js:37]: https://github.com/testing-library/testing-playground/pull/56#discussion_r432099970

File: src/lambda/oembed/oembed.js [hunk 2]
```diff
@@ -34,7 +34,8 @@ function handler(event, context, callback) {
...
+  const iframeSrc = url.replace('gist', 'embed');
```
Comment (smeijer) [inline src/lambda/oembed/oembed.js:38]: Should we also provide a default set of panes? All 4 panes on 900 pixels is a bit much?

File: src/lambda/oembed/oembed.js (hunk 1 above)
Comment (MichaelDeBoey) [reply src/lambda/oembed/oembed.js:37]: Totally forgot about that one 😓

File: src/lambda/oembed/oembed.js (hunk 2 above)
Comment (MichaelDeBoey) [reply src/lambda/oembed/oembed.js:38]: I'll add `preview` & `result`, the defaults that are used in the GUI

Comment (smeijer) [reply src/lambda/oembed/oembed.js:38]: Not entirely sure about this though. The user can customize the url / embed code via the embed frame if be wants to. And it's hard to decide what are the right frames to show, as that can depend on the situation.

If we choose a default, I vote for `markup,preview,query`. What do you think?

Comment (MichaelDeBoey) [reply src/lambda/oembed/oembed.js:38]: Then I think we should make that the default in the GUI too to be consistent

**Review by smeijer — APPROVED** [review]
Thanks. Looking good.

File: src/lambda/oembed/oembed.js (hunk 2 above)
Comment (smeijer) [reply src/lambda/oembed/oembed.js]: Ok. Let's merge this, and look at that another time 🙂

**Pull request merged**